connectors/concur/src/test/resources/concur/responses/report_id_1.json

```diff
@@ -1,193 +1,309 @@
-    "Code": "string",
-    "ListItemID": "string",
-    "Type": "string",
-    "Value": "string"
-  },
-  "Custom20": {
-    "Code": "string",
-    "ListItemID": "string",
-    "Type": "string",
-    "Value": "string"
-  },
-  "Custom3": {
-    "Code": "string",
-    "ListItemID": "string",
-    "Type": "string",
-    "Value": "string"
-  },
-  "Custom4": {
-    "Code": "string",
-    "ListItemID": "string",
-    "Type": "string",
-    "Value": "string"
-  },
-  "Custom5": {
-    "Code": "string",
-    "ListItemID": "string",
-    "Type": "string",
-    "Value": "string"
-  },
-  "Custom6": {
-    "Code": "string",
-    "ListItemID": "string",
-    "Type": "string",
-    "Value": "string"
-  },
-  "Custom7": {
-    "Code": "string",
-    "ListItemID": "string",
-    "Type": "string",
-    "Value": "string"
-  },
-  "Custom8": {
-    "Code": "string",
-    "ListItemID": "string",
-    "Type": "string",
-    "Value": "string"
-  },
-  "Custom9": {
-    "Code": "string",
-    "ListItemID": "string",
-    "Type": "string",
-    "Value": "string"
-  },
-  "EverSentBack": true,
-  "HasException": true,
-  "ID": "79D89435DAE94F53BF60",
-  "LastComment": "string",
-  "LastModifiedDate": "2017-11-28T04:14:05.918Z",
-  "LedgerName": "Starbucks",
-  "Name": "string",
-  "OrgUnit1": {
-    "Code": "string",
-    "ListItemID": "string",
-    "Type": "string",
-    "Value": "string"
-  },
-  "OrgUnit2": {
-    "Code": "string",
-    "ListItemID": "string",
-    "Type": "string",
-    "Value": "string"
-  },
-  "OrgUnit3": {
-    "Code": "string",
-    "ListItemID": "string",
-    "Type": "string",
-    "Value": "string"
-  },
-  "OrgUnit4": {
-    "Code": "string",
-    "ListItemID": "string",
-    "Type": "string",
-    "Value": "string"
-  },
-  "OrgUnit5": {
-    "Code": "string",
-    "ListItemID": "string",
-    "Type": "string",
-    "Value": "string"
+    "Type": "ConnectedList",
+    "Value": "None",
+    "Code": "None"
   },
-  "OrgUnit6": {
-    "Code": "string",
-    "ListItemID": "string",
-    "Type": "string",
-    "Value": "string"
+  "Custom3": {},
+  "Custom4": {},
+  "Custom5": {},
+  "Custom6": {},
+  "Custom7": {},
+  "Custom8": {},
+  "Custom9": {},
+  "Custom10": {},
+  "Custom11": {},
+  "Custom12": {},
+  "Custom13": {},
+  "Custom14": {},
+  "Custom15": {},
+  "Custom16": {},
+  "Custom17": {},
+  "Custom18": {},
+  "Custom19": {},
+  "Custom20": {},
+  "LedgerName": "ORACLE",
+  "PolicyID": "gWk3QXwsDWJoW5Yar$ph$sJI$pPLdwGyGTM71g",
+  "EverSentBack": "N",
+  "ReportImageURL": "https://imagingupload.concursolutions.com/file/t0030425xl9t/52C2430FA25414D35216B02E93D2C8F5011FA306E2CBCFFEEE2A53653F4A0F9C26BA34BAC9FC8CFCA12F323D75B82A90246F63CFDB3C1CDCF1573E06E6E0B92FH722CCBEF3F16D8B8C7743AC35E44941D?id=7BCBC522BBF94102B368&e=t0030425xl9t&t=AN",
+  "HasException": "N",
+  "WorkflowActionURL": "https://implementation.concursolutions.com/api/expense/expensereport/v1.1/report/gWujNPAb67r9LjhqgN7BEYYaQOWzavXBtUP1sej$sXfPQ/WorkFlowAction",
```

Comment:
Is this URL from a real response?  The WorkFlowAction after the reportId in the URL isn't mentioned in their docs: https://developer.concur.com/api-reference-deprecated/version-one-one/expense-report/expense-report-header-post.html

Reply:
The above URL is not the correct documentation. Documentation link for approving/rejecting the workflowaction is "https://developer.concur.com/api-reference/expense/expense-report/post-report-workflow-action.html"